scripts/run-scss-lint.sh

```diff
@@ -4,6 +4,7 @@
 
 PROJECT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd)
 
-docker run --rm \
-  -v $PROJECT_DIR:/style-guide/ \
-  brainly/style-guide scss-lint "$@"
+docker run -ti --rm \
+  -v $PROJECT_DIR/src:/style-guide/src \
```

Comment:
Updated mounting - we were including node_modules before and it was causing errors!